app/src/main/java/com/babylon/wallet/android/data/dapp/model/SubintentExpiration.kt

```diff
@@ -23,10 +18,7 @@ sealed interface SubintentExpiration {
             DappToWalletInteractionSubintentExpireAtTime(unixTimestampSeconds = expireAt.inWholeSeconds.toULong())
         )
 
-        override fun expirationTimestamp(): Timestamp =
-            Timestamp.ofInstant(Instant.ofEpochMilli(expireAt.inWholeMilliseconds), ZoneId.systemDefault())
-
-        fun expirationDuration(): Duration = (expireAt.inWholeSeconds - Instant.now().epochSecond).coerceAtLeast(0L).seconds
+        fun expirationDuration(): Duration = (expireAt.inWholeSeconds - java.time.Instant.now().epochSecond).coerceAtLeast(0L).seconds
```

Comment:
I think you can insert the package?

Reply:
What do you mean? Import? It conflicts with the `Instant` type from Sargon